buildcomponents.yaml

```diff
@@ -0,0 +1,13 @@
+.migrate:
+  image: kevinrambaud/node-flyway:8-5.1.4-alpine
+  script:
+    - cp -r $CI_PROJECT_DIR/sql/* /flyway/sql/
+    - sh /flyway/flyway -url=jdbc:postgresql://$POSTGRES_SERVER:5432/$POSTGRES_DB -schemas=public -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD migrate
+
+.deploy:
+  script:
+    - grunt deploy --environment=$DEPLOYENV --packageVersion=$VERSION --packageName=$CI_PROJECT_NAME
```

Comment:
Instead of `DEPLOYENV`, consider [`CI_ENVIRONMENT_NAME`](https://docs.gitlab.com/ee/ci/variables/#predefined-environment-variables). That is a gitlab feature and other things in the gitlab UI will hook into that nicely.

Reply:
We do that in our [java template](https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/java-service.gitlab-ci.yml), would be nice to do it in the node one too. Actually, with this, we can share the deploy job between java and node.